docs/articles_en/about-openvino/release-notes-openvino/system-requirements.rst

```diff
@@ -162,6 +167,208 @@ Operating systems and developer environment
 
    OpenVINO Python binaries are built with and redistribute oneTBB libraries.
 
+OpenVINO Distributions
+######################
+
+.. tab-set::
+
+   .. tab-item:: Archive
+      :name: archive-sysreq
+
+      .. tab-set::
+
+         .. tab-item:: Linux
+            :name: archive-lnx-sysreq
+
+            * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`__
+            * `Python 3.9 - 3.12, 64-bit <https://www.python.org/downloads/>`__
+            * GCC:
+
+            .. tab-set::
+
+               .. tab-item:: Ubuntu 20.04
+                  :sync: ubuntu-20
```

Comment:
should we also cover Ubuntu 22 and 24 ?

Reply:
I have applied your suggestions in:

https://github.com/openvinotoolkit/openvino/pull/27233/commits/f0dcf76a28c43b3943e2687506339e9c585a9a85